app/views/facility_journals/_problem_transactions.html.haml

```diff
@@ -0,0 +1,68 @@
+.row
+  .span12
+    %p.alert.alert-error= t("facility_journals.new.expired_payment")
+.row
+  .span12
+    = will_paginate(order_details) if order_details.respond_to? :total_pages
+    %table.table.table-striped.table-hover.dense.old-table
+      %thead
+        %tr
+          %th= Order.model_name.human
+          %th= OrderDetail.model_name.human
+          %th.nowrap= OrderDetail.human_attribute_name(@date_range_field)
+          - if @extra_date_column
+            %th.nowrap= OrderDetail.human_attribute_name(@extra_date_column_header || @extra_date_column)
+          - if current_facility.blank? || cross_facility?
+            %th= Facility.model_name.human
+          %th.nowrap= [Account.model_name.human, Account.human_attribute_name("expires_at")].join(" ")
+          %th.order-note= OrderDetail.human_attribute_name("description")
+          %th.nowrap= OrderDetail.human_attribute_name("user")
+          - unless @account
+            %th= Account.model_name.human
+            %th= Account.human_attribute_name("owner")
+          %th.currency= OrderDetail.human_attribute_name("cost")
+          %th= OrderDetail.human_attribute_name("order_status")
+          - if local_assigns[:show_statements]
```

Comment:
I don't think this will ever be true in this scenario. Should probably take it out.